prov/efa/src/efa_hmem.c

```diff
@@ -135,11 +136,12 @@ static int efa_hmem_info_init_cuda(struct efa_hmem_info *cuda_info, struct efa_d
 		return -FI_ENOMEM;
 	}
 
-	/*
-	 * TODO set to false when non-p2p support is added, or set conditionally
-	 * based on some environment variable
-	 */
+	/* TODO query FI_HMEM_CUDA_ENABLE_XFER and set conditionally */
```

Comment:
Why would it depend on FI_HMEM_CUDA_ENABLE_XFER? Maybe you meant `FI_HMEM_P2P_DISABLED`?

Reply:
I'll remove the comment